doc/install.rst

```diff
@@ -66,9 +67,38 @@ Installing the latest release
             python -m pip freeze             # show all installed packages in the environment
             python -c "import sklearn; sklearn.show_versions()"
 
-        .. tab-item:: macOS
-          :class-label: tab-4
-          :sync: os-macos
+        .. tab-item:: conda
+          :class-label: tab-6
+          :sync: packager-conda
+
+          Install conda using the `Anaconda or miniconda installers
```

Comment:
Maybe try to use substitution to reuse the same conda content in multiple places: https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#substitutions

Not sure if that works with complex multi-line content, if not [Including a file](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/reuse.html#include-a-shared-file) may be an option?